gem/lib/earlgrey/cli.rb

```diff
@@ -19,11 +19,12 @@ class CLI < Thor
     package_name 'EarlGrey'
 
     desc 'install', 'Installs EarlGrey into an Xcode unit test target'
+    method_option :carthage, type: :boolean, default: true
     method_option :project, aliases: '-p', type: :string, required: false, desc: 'Project'
-    method_option :target,  aliases: '-t', type: :string, required: true, desc: 'EarlGreyTestTarget'
     method_option :scheme,  aliases: '-s', type: :string, required: false, desc: 'EarlGreyTestTarget.xcscheme'
     method_option :swift,    type: :boolean, default: true
-    method_option :carthage, type: :boolean, default: true
+    method_option :swift_version, type: :numeric, default: 3.0, :enum => %w(2.2, 3.0)
```

Comment:
```ruby
%w(2.2, 3.0)
```

The above creates a corrupted first version value. Observe:

```
> %w(2.2, 3.0)
=> ["2.2,", "3.0"]
```

There shouldn't be a comma in element 0. The space delimiter is what's used, not a comma.